### PR TITLE
Disabling account registration for preview and prod profile

### PR DESCRIPTION
**Pull request by DanielW1987**

### CODE REVIEW

**Comment by DanielW1987** [comment]
[Deactivate registration temporarily](https://trello.com/c/C0cWEZ7M/148-deactivate-registration-temporarily)

File: webapp/src/test/java/rocks/metaldetector/web/controller/mvc/authentication/PreviewRegistrationControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  private PreviewRegistrationController underTest = new PreviewRegistrationController();
+  private RestAssuredMockMvcUtils restAssuredUtils;
+
+  @BeforeEach
+  void setup() {
+    restAssuredUtils = new RestAssuredMockMvcUtils(Endpoints.Guest.REGISTER);
+    RestAssuredMockMvc.standaloneSetup(
+            underTest,
+            springSecurity((request, response, chain) -> chain.doFilter(request, response)),
+            exceptionResolver()
```
Comment (hedgehog1833) [inline webapp/src/test/java/rocks/metaldetector/web/controller/mvc/authentication/PreviewRegistrationControllerTest.java:31]: The exception resolver is not needed when not tests with exception exist. Please also remove the interface above.

File: webapp/src/main/resources/templates/guest/auth/disabled-register.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      </div>
+
+      <!-- Verify Password -->
+      <div class="form-label-group">
+        <input type="password" id="verifyPlainPassword" class="form-control" name="verifyPlainPassword" disabled placeholder="" autocomplete="new-password">
+        <label for="verifyPlainPassword">Verify Password</label>
+      </div>
+
+      <!-- Register -->
+      <button class="btn btn-lg btn-dark btn-block mt-4" type="submit" disabled>Join Metal Detector</button>
```
Comment (hedgehog1833) [inline webapp/src/main/resources/templates/guest/auth/disabled-register.html:36]: I would just leave this out. Why should we have the fields if noboby can register? Just the text and the already registered stuff.

File: webapp/src/test/java/rocks/metaldetector/web/controller/mvc/authentication/PreviewRegistrationControllerTest.java (hunk 1 above)
Comment (DanielW1987) [reply webapp/src/test/java/rocks/metaldetector/web/controller/mvc/authentication/PreviewRegistrationControllerTest.java:31]: The `SecurityMockMvcConfigurer` is needed.

File: webapp/src/main/resources/templates/guest/auth/disabled-register.html (hunk 2 above)
Comment (DanielW1987) [reply webapp/src/main/resources/templates/guest/auth/disabled-register.html:36]: Only the narrow text does not look good. I had looked at that first and then decided on the variants with the deactivated fields.

**Pull request merged**